pkgs/data/fonts/babelstone-han/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenvNoCC.mkDerivation {
   pname = "babelstone-han";
-  version = "13.0.3";
+  version = "16.0.2";
 
   src = fetchzip {
     # upstream download links are unversioned, so hash changes
-    url = "https://web.archive.org/web/20200210125314/https://www.babelstone.co.uk/Fonts/Download/BabelStoneHan.zip";
-    hash = "sha256-LLhNtHu2hh5DY2XVSrLuVzzR6OtMdSSHetyA0k1IFs0=";
+    url = "https://www.babelstone.co.uk/Fonts/Download/BabelStoneHan.zip";
```

Comment:
We historically had to use an archive link here since the URL is unstable. It looks like upstream now has versioned URLs like https://www.babelstone.co.uk/Fonts/Download/BabelStoneHan-16.0.2.zip, but that only the last two versions are kept. So I think we should still probably wait a little bit here for the Wayback Machine to stop being read‐only to use an archived link again here.